buildout.cfg

```diff
@@ -1,6 +1,7 @@
 [buildout]
 develop = .
 parts = test scripts
+show-picked-versions = true
```

Comment:
Is this a leftover diagnostic configuration?

Reply:
I intentionally left it there for future diagnosis: `pip` shows the versions it has installed buildout does not by default and we do not use version pins here.